source/renderer/app/components/staking/rewards/StakingRewardsForIncentivizedTestnet.js

```diff
@@ -326,6 +380,46 @@ export default class StakingRewardsForIncentivizedTestnet extends Component<
                                   <LoadingSpinner medium />
                                 </PopOver>
                               </div>
+                            ) : (
+                              <>
+                                {rewardsAddress && (
+                                  <div>
+                                    <CopyToClipboard
+                                      text={rewardsAddress}
+                                      onCopy={() =>
+                                        onCopyAddress(rewardsAddress)
+                                      }
+                                    >
+                                      <div className={styles.addressContainer}>
+                                        <span className={styles.address}>
+                                          {ellipsis(rewardsAddress, 15, 15)}
+                                        </span>
+                                        <span className={styles.copyAddress}>
+                                          <SVGInline
+                                            svg={iconCopy}
+                                            className={styles.copyIcon}
+                                          />
+                                        </span>
+                                      </div>
+                                    </CopyToClipboard>
+                                    {/* <ButtonLink
```

Comment:
@aleksandardjordjeviciohk please set a const in this component to be used to enable/disable the explorer link button instead of commenting this out - e.g. `const IS_EXPLORER_LINK_BUTTON_ENABLED = false;` 

Please rename the "learnMoreButton" into "explorerButton".

Reply:
Updated